src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltObservationTable.js

```diff
@@ -311,7 +262,10 @@ const FishBeltObservationTable = ({
         })
       }
 
-      const proposeNewSpeciesClick = () => openNewObservationModal(observationId)
+      const proposeNewSpeciesClick = () => {
+        setObservationIdToAddNewBenthicAttributeTo(observationId)
```

Comment:
needs rename. we are dealing with fishName not benthic attribute, but that name change would need to happen all over cause its needed in the 'alternate' abstraction too.